### ISSUE TITLE
[documentation] What is the difference between send and sendTransaction?

### ISSUE THREAD

**Issue by JustinDrake** [trufflesuite/truffle#474]
In my JS test code, an instance of a deployed contract has the functions `send` and `sendTransaction` associated with it. It seems that both behave similarly, calling the contract's default function. How are `send` and `sendTransaction` different?

I [could not find](https://readthedocs.org/search/?q=sendTransaction&check_keywords=yes&area=default&project=truffle&version=beta&type=file) `sendTransaction` in the documentation. I'd be happy to submit a PR to the docs when this is cleared up. 👍

**Comment by tcoulter** [comment]
Hi there. Both are listed in our documentation here: http://truffleframework.com/docs/getting_started/contracts#sending-ether-to-a-contract

TL;DR: `send` is just shorthand if you only want to send Ether and don't want to call a specific contract function. `sendTransaction` will give you more options for sending the transaction, and you do have the ability to call contract functions other than the default.

I'm going to close this since it is documented. Let me know if you have any questions.
